index.js

```diff
@@ -1,17 +1,42 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length++;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(!this.items[pos]) throw new Error('OutOfBounds');
+    return this.items[pos]; 
+  }
 
-  max() {}
+  max() {
+    if (!this.length) throw new Error('EmptySortedList');
+    return this.items[this.items.length - 1];
+  }
 
-  min() {}
+  min() {
+    if (!this.length) throw new Error('EmptySortedList');
+    return this.items[0];
+  }
 
-  sum() {}
+  sum() {
+    const sum = this.items.reduce((a, b) => a + b, 0);
+    return sum;
+  }
 
-  avg() {}
+  avg() {
+    if(!this.length) {
+      throw new Error('EmptySortedList');
+    }
+    const avg = this.items.reduce((a, b) => a + b, 0) / this.length;
```

Comment:
if an average is "sum over length", you could use the sum method. This also work, but you have code repetition